src/components/asthma/components/AsthmaProviderReport/AsthmaProviderReport.tsx

```diff
@@ -247,137 +247,123 @@ export default function (props: AsthmaProviderReportProps) {
+                    <div style={{ flexGrow: 1, flexBasis: '30%', backgroundColor: '#f2f2f2', border: '1px solid #dbdbdb', borderRadius: '10px', padding: '12px' }}>
+                        <div style={{ marginBottom: '8px', fontWeight: 700 }}>{language('asthma-provider-report-top-triggers')}</div>
+                        {renderStat(language('asthma-provider-report-animal-exposure'), animalExposureDays, animalExposureDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))}
+                        {renderStat(language('asthma-provider-report-allergens'), allergenDays, allergenDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))}
+                        {renderStat(language('asthma-provider-report-smoke'), smokeDays, smokeDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))}
+                        {renderStat(language('asthma-provider-report-air-pollution'), airPollutionDays, airPollutionDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))}</div>
+                    <div style={{ flexGrow: 1, flexBasis: '30%', backgroundColor: '#f2f2f2', border: '1px solid #dbdbdb', borderRadius: '10px', padding: '12px', margin: '0 16px' }}>
+                        <div style={{ marginBottom: '8px', fontWeight: 700 }}>{language('asthma-provider-report-medications')}</div>
+                        {renderStat(language('asthma-provider-report-missed-doses'), missedDosesDays, missedDosesDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))}
+                        {barrierDays === 0 && renderStat(language('asthma-provider-report-barriers-adherence'), 0, language('asthma-provider-report-reports'))}
+                        {barrierDays >= 1 && renderStat(language('asthma-provider-report-barriers-adherence-with-asterisk'), barrierDays, barrierDays === 1 ? language('asthma-provider-report-report') : language('asthma-provider-report-reports'))} {barrierDays >= 1 &&
+                            <div style={{ fontSize: '12px', marginTop: '12px' }}>*{barrierAnswers.join('; ')}</div>
                         }
                     </div>
-                    <div style={{flexGrow: 1, flexBasis: '30%', backgroundColor: '#f2f2f2', border: '1px solid #dbdbdb', borderRadius: '10px', padding: '12px'}}>
-                        <div style={{marginBottom: '8px', fontWeight: 700}}>Days with AQI over 100</div>
-                        {renderStat('Home', highHomeAqiDays > 0 ? highHomeAqiDays : 'None', highHomeAqiDays === 1 ? 'day' : highHomeAqiDays > 1 ? 'days' : '')}
-                        {renderStat('Work', highWorkAqiDays > 0 ? highWorkAqiDays : 'None', highWorkAqiDays === 1 ? 'day' : highWorkAqiDays > 1 ? 'days' : '')}
```

Comment:
This line should not have been removed.  The duplicate is down below where indicated.  Line 344, I think, where it says "Source: airnow.gov" in plain text.